db/seeds/api.controller.js

```diff
@@ -54,3 +55,19 @@ exports.getArticleById = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.getComments = (req, res, next) => {
+  const { article_id } = req.params;
+  selectComments(article_id)
+    .then((comments) => {
+      console.log(comments, "<--article in controller");
+      if (comments.length === 0 || !comments) {
+        const error = new Error("Not Found");
```

Comment:
Is it an error if an article has no comments?? What about a new article, it wouldn't have any new comments as no one could have commented on it.